modules/ROOT/pages/salesforce/salesforce-connector-910.adoc

```diff
@@ -0,0 +1,200 @@
+= Salesforce Connector 9.10 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Salesforce (Salesforce Connector) enables you to create apps that react to Salesforce events such as adding, changing, or deleting objects, topics, documents, and channels.
+
+Salesforce Connector enables you to connect to the Salesforce APIs. This connector exposes methods for accessing Salesforce, including working with the https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_dev_process_chapter.htm[Salesforce Apex classes].
+
+This connector works with the Salesforce SOAP API, REST API, Bulk API, and Streaming API, depending on the operation you configure. Each API call uses an XML request and response over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
+
+Release Notes: xref:release-notes::connector/salesforce-connector-release-notes-mule-4.adoc[Salesforce Connector Release Notes] +
```

Comment:
I've been adding this text in front of the release note link: 
For software requirements and compatibility information, see the...
I have also been moving the Exchange link under See Also and the reference link under See Also.